viewer/whylogs-script.js

```diff
@@ -0,0 +1,547 @@
+"use strict";
+
+(function () {
+  var MESSAGES = {
+    error: {
+      noInputElementFound: "It seems we could not find the file input element.",
+      noBrowserSupport: "This browser does not seem to support the `files` property of file inputs.",
+      noFileSelected: "Please select a file.",
+      fileAPINotSupported: "The file API is not supported on this browser yet.",
+      invalidJSONFile:
+        "It seems the JSON file you are trying to load is not valid due because it does not match whylogs format. Please check the file validity and try again.",
+    },
+  };
+
+  // HTML Elements
+  var $selectedProfile = $(".wl__selected-profile");
```

Comment:
There is no specific reason for that. I started writing code with ES5 because we are not using any compiler/transpiler and I wanted to have a greater browser support. However, every required browser now supports ES6 by default, so it was my silly mistake.

I would probably recommend a quick refactoring to ES6, because we are mixing arrow functions and regular functions as well. Just to be more consistent.